src/System.Text.Json/tests/Utf8JsonWriterTests.cs

```diff
@@ -2311,6 +2311,57 @@ public void WriteBase64StringEscaped(bool formatted, bool skipValidation)
             Assert.True(Encoding.UTF8.GetBytes(propertyName).AsSpan().SequenceEqual(propertyNameSpanUtf8));
         }
 
+        [Theory]
+        [InlineData(true, true)]
+        [InlineData(true, false)]
+        [InlineData(false, true)]
+        [InlineData(false, false)]
+        public void WriteBase64StringEscapeLengthBigger(bool formatted, bool skipValidation)
```

Comment:
nit: **If feasible,** consider instead combining this test case with the ones above, into a single `WriteBase64String` test, similar to `WriteDateTimesValue`. So, instead of having 3 separate tests that primarily just defer on the property name, we have 1 test, with different inline data.

https://github.com/dotnet/corefx/blob/063e38c7bc66e076fdcc7521e4c69650baae9427/src/System.Text.Json/tests/Utf8JsonWriterTests.cs#L2206-L2312

https://github.com/dotnet/corefx/blob/063e38c7bc66e076fdcc7521e4c69650baae9427/src/System.Text.Json/tests/Utf8JsonWriterTests.cs#L4255-L4268

Reply:
Isn't it better to have separate methods which names illustrates what exactly testing, I would prefer to move shared/similar code section into a method but leave separate test methods

Reply:
Combined tests instead of keeping separate, updated string values to illustrate what it is testing